test/integration/correctness/application.c

```diff
@@ -6,7 +6,7 @@
  * SPDX-FileCopyrightText: 2008-2023 HPDCS Group <rootsim@googlegroups.com>
  * SPDX-License-Identifier: GPL-3.0-only
  */
-#include <tests/integration/correctness/application.h>
+#include "application.h"
```

Comment:
On this, I have doubts: this is not on the include path, because it's not a "project" header, but an "application" header, although it is a test. In this case, I would use quotes. What's your opinion?

Reply:
In this case, the full path with the angled brackets is not aesthetically pleasing, so it's fine.